ibis/pandas/aggcontext.py

```diff
@@ -313,6 +314,92 @@ def compute_window_spec_interval(_, expr):
     return pd.tseries.frequencies.to_offset(value)
 
 
+def _window_agg_built_in(
+    frame, windowed, function, max_lookback, *args, **kwargs
+):
+    """Apply window aggregation with built-in aggregators.
+    """
+    assert isinstance(function, str)
+    method = operator.methodcaller(function, *args, **kwargs)
+
+    if max_lookback is not None:
+        agg_method = method
+
+        def sliced_agg(s):
+            return agg_method(s.iloc[-max_lookback:])
```

Comment:
can this not just be method?

Reply:
I got a weird exception when changing to `method` causes error (error message: sliced_agg() got an unnexpected keyword argument 'raw'). I am not quite sure what's going on with nested method caller.